addons/TBMod_arsenal/arsenal/fn_arsenalRUSS.sqf

```diff
@@ -488,6 +485,29 @@ _items append (switch (ACE_player getVariable ["TB_rolle", ""]) do
         _allgemein_kopfbedeckung +
         _gepaeck_mittel +
         [
+            // Gewehre
+            "rhs_weap_svdp",
+            "rhs_weap_svdp_wd",
+
+            "rhs_weap_akms",         // 7,62 keine Visierung möglich
+            "rhs_30Rnd_762x39mm_polymer",
```

Comment:
Keine Munition in die Klassen rein, sonst kann der Muniträger das nicht mitnehmen... IMMER in die Mengen, sonst eine neue machen.